CL/cl_ext.h

```diff
@@ -4294,6 +4294,36 @@ clCancelCommandsIMG(
 
 #endif /* !defined(CL_NO_NON_ICD_DISPATCH_EXTENSION_PROTOTYPES) */
 
+/*************************************
+ * cl_qcom_perf_hint extension *
```

Comment:
Is there a matching PR that adds this to the https://github.com/KhronosGroup/OpenCL-Docs/blob/main/xml/cl.xml file? Otherwise we'll need to maintain this diff whenever the headers are automatically generated from the xml.

This is also missing things that would automatically get generated from the xml, like the macros for the extension name and extension version.